src/nvim/os/win_defs.h

```diff
@@ -4,6 +4,7 @@
 #include <windows.h>
 #include <sys/stat.h>
 #include <stdio.h>
+#include <uv.h>
```

Comment:
Like was done in `unix_defs.h`, could you mention the reason this is needed in a comment?

Even something small like this would be good, IMO:
```
#include <uv.h>  /* S_IFLNK */
```